src/mesh/boundary_standard.cxx

```diff
@@ -23,88 +23,92 @@
     lead to an out of bounds access error later but we add it here to provide a
     more explanatory message.
  */
-void verifyNumPoints(BoundaryRegion *region, int ptsRequired) {
-  TRACE("Verifying number of points available for BC");
+namespace {
+  void verifyNumPoints(BoundaryRegion *region, int ptsRequired) {
```

Comment:
I think you've just added this to an anonymous namespace and indented it? Github is not very helpful, even with whitespace changes hidden

Reply:
yep, that's all I did.

Reply:
(and replaced mesh with localmesh!)